acai-ios/CollectionViewCells/HeaderImageCollectionViewCell.swift

```diff
@@ -18,7 +18,7 @@ class HeaderImageCollectionViewCell: UICollectionViewCell {
     private enum FileConstants {
         static let imageViewHeightWidth: CGFloat = 206
     }
-
+    
```

Comment:
delete white spaces in this file. make sure you have `trim whitespaces` in `Preferences`
<img width="799" alt="Screen Shot 2019-04-10 at 9 50 35 PM" src="https://user-images.githubusercontent.com/26048121/55925152-b8076000-5bda-11e9-9cc8-6c1ba1e48aba.png">